src/popup/router/components/Header.vue

```diff
@@ -1,49 +1,51 @@
 <template>
   <div class="header" v-if="showNavigation && !aeppPopup">
-    <div class="content">
-      <div class="left">
-        <Logo class="logo" v-if="isLoggedIn" />
-        <button v-if="title && !tourRunning" @click="back" class="icon-btn back">
-          <Back data-cy="back-arrow" />
-        </button>
-      </div>
-
-      <div class="title">
-        <TruncateMid
-          :str="pageTitle || (title && $t(`pages.titles.${title}`)) || $t('pages.titles.home')"
-          class="text"
-        />
-      </div>
-
-      <div class="right">
-        <template v-if="isLoggedIn">
-          <span
-            v-if="!$route.path.startsWith('/notifications')"
-            @click="toNotifications"
-            class="notifications"
-            data-cy="noti"
-          >
-            <button class="icon-btn">
-              <Bell />
-            </button>
-            <span v-if="notificationsCount" class="badge" data-cy="noti-count">
-              {{ notificationsCount }}
-            </span>
-          </span>
-
-          <button
-            v-if="$route.path === '/notifications'"
-            @click="$router.push('/notifications/settings')"
-            class="icon-btn settings"
-          >
-            <Settings />
-          </button>
-
-          <button @click="$emit('toggle-sidebar')" class="icon-btn">
-            <Menu data-cy="hamburger" />
-          </button>
-        </template>
-      </div>
+    <div v-if="isLoggedIn || (title && !tourRunning)" class="left">
+      <RouterLink
+        v-if="isLoggedIn"
+        to="/account"
+        :disabled="$route.path === '/account'"
```

Comment:
I"m not sure that `disabled` works on RouterLink. Actually, seems that this check is not necessary.